<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BuildingEditorWindow</class>
 <widget class="QMainWindow" name="BuildingEditorWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Building Editor</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <property name="margin">
     <number>0</number>
    </property>
    <item>
     <widget class="BuildingEditor::FloorView" name="floorView"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="statusBarLayout">
      <item>
       <widget class="QLabel" name="coordLabel">
        <property name="text">
         <string>999,999</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QComboBox" name="editorScaleComboBox"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNewBuilding"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSaveAs"/>
    <addaction name="separator"/>
    <addaction name="actionExportTMX"/>
    <addaction name="separator"/>
    <addaction name="actionClose"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionPreferences"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionZoomIn"/>
    <addaction name="actionZoomOut"/>
    <addaction name="actionNormalSize"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuView"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionPecil"/>
   <addaction name="actionEraser"/>
   <addaction name="actionDoor"/>
   <addaction name="actionWindow"/>
   <addaction name="actionStairs"/>
   <addaction name="actionSelectObject"/>
   <addaction name="actionUpLevel"/>
   <addaction name="actionDownLevel"/>
  </widget>
  <widget class="QDockWidget" name="dockWidget">
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QToolBox" name="toolBox">
       <property name="minimumSize">
        <size>
         <width>192</width>
         <height>0</height>
        </size>
       </property>
       <widget class="QWidget" name="page">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>192</width>
          <height>450</height>
         </rect>
        </property>
        <attribute name="label">
         <string>Page 1</string>
        </attribute>
       </widget>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QComboBox" name="scaleComboBox">
         <property name="frame">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open...</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="text">
    <string>Undo</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="text">
    <string>Redo</string>
   </property>
  </action>
  <action name="actionNewBuilding">
   <property name="text">
    <string>New...</string>
   </property>
  </action>
  <action name="actionPecil">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="buildingeditor.qrc">
     <normaloff>:/BuildingEditor/icons/icon_pencil.png</normaloff>:/BuildingEditor/icons/icon_pencil.png</iconset>
   </property>
   <property name="text">
    <string>Pecil</string>
   </property>
   <property name="toolTip">
    <string>Draw Room</string>
   </property>
  </action>
  <action name="actionEraser">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="buildingeditor.qrc">
     <normaloff>:/BuildingEditor/icons/icon_eraser.png</normaloff>:/BuildingEditor/icons/icon_eraser.png</iconset>
   </property>
   <property name="text">
    <string>Eraser</string>
   </property>
   <property name="toolTip">
    <string>Erase Room</string>
   </property>
  </action>
  <action name="actionDoor">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="buildingeditor.qrc">
     <normaloff>:/BuildingEditor/icons/icon_door.png</normaloff>:/BuildingEditor/icons/icon_door.png</iconset>
   </property>
   <property name="text">
    <string>Door</string>
   </property>
   <property name="toolTip">
    <string>Place Door</string>
   </property>
  </action>
  <action name="actionWindow">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="buildingeditor.qrc">
     <normaloff>:/BuildingEditor/icons/icon_window.png</normaloff>:/BuildingEditor/icons/icon_window.png</iconset>
   </property>
   <property name="text">
    <string>Window</string>
   </property>
   <property name="toolTip">
    <string>Place Window</string>
   </property>
  </action>
  <action name="actionStairs">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="buildingeditor.qrc">
     <normaloff>:/BuildingEditor/icons/icon_stair.png</normaloff>:/BuildingEditor/icons/icon_stair.png</iconset>
   </property>
   <property name="text">
    <string>Stairs</string>
   </property>
   <property name="toolTip">
    <string>Place Stairs</string>
   </property>
  </action>
  <action name="actionUpLevel">
   <property name="icon">
    <iconset resource="buildingeditor.qrc">
     <normaloff>:/BuildingEditor/icons/icon_floorup.png</normaloff>:/BuildingEditor/icons/icon_floorup.png</iconset>
   </property>
   <property name="text">
    <string>Up One Level</string>
   </property>
  </action>
  <action name="actionDownLevel">
   <property name="icon">
    <iconset resource="buildingeditor.qrc">
     <normaloff>:/BuildingEditor/icons/icon_floordn.png</normaloff>:/BuildingEditor/icons/icon_floordn.png</iconset>
   </property>
   <property name="text">
    <string>Down One Level</string>
   </property>
  </action>
  <action name="actionSelectObject">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="buildingeditor.qrc">
     <normaloff>:/BuildingEditor/icons/icon_select.png</normaloff>:/BuildingEditor/icons/icon_select.png</iconset>
   </property>
   <property name="text">
    <string>Select Objects</string>
   </property>
  </action>
  <action name="actionExportTMX">
   <property name="text">
    <string>Export TMX...</string>
   </property>
  </action>
  <action name="actionZoomIn">
   <property name="icon">
    <iconset resource="../tiled.qrc">
     <normaloff>:/images/16x16/zoom-in.png</normaloff>:/images/16x16/zoom-in.png</iconset>
   </property>
   <property name="text">
    <string>Zoom In</string>
   </property>
  </action>
  <action name="actionZoomOut">
   <property name="icon">
    <iconset resource="../tiled.qrc">
     <normaloff>:/images/16x16/zoom-out.png</normaloff>:/images/16x16/zoom-out.png</iconset>
   </property>
   <property name="text">
    <string>Zoom Out</string>
   </property>
  </action>
  <action name="actionNormalSize">
   <property name="icon">
    <iconset resource="../tiled.qrc">
     <normaloff>:/images/16x16/zoom-original.png</normaloff>:/images/16x16/zoom-original.png</iconset>
   </property>
   <property name="text">
    <string>Normal Size</string>
   </property>
  </action>
  <action name="actionPreferences">
   <property name="text">
    <string>Preferences...</string>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="text">
    <string>Save As...</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>BuildingEditor::FloorView</class>
   <extends>QGraphicsView</extends>
   <header>BuildingEditor/FloorEditor.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../tiled.qrc"/>
  <include location="buildingeditor.qrc"/>
 </resources>
 <connections/>
</ui>
